dino.icu.yaml

```
@@ -76,6 +76,11 @@ geta: # yoda - https://getadinoicu.yodacode.repl.co/dino.png - get a dino from h
   type: CNAME
   value: ef35bb48-f121-4f7f-91f6-93f601559c24.id.repl.co.
 
+icons: # yoda - https://github.com/yodalightsabr/icons-dino-icu - get a dino from hackclub/dinosaurs api
```

```suggestion
icons: # yoda - https://github.com/yodalightsabr/icons-dino-icu - a CDN for @hackclub/icons
```

Small thing, I think you forgot to edit the comment.

---

Oops- thanks!